### ISSUE TITLE
Add pluralize for table name

### ISSUE THREAD

**Issue by listepo** [balderdashy/sails#2918]
I think pluralize for table name is good feature.

That table is called ```transactions``` need to write in the model ```Transaction.js``` 
```
...
tableName: transactions
...
```
But this is not practical.
Make it possible to enable or disable this feature.

**Comment by tjwebb** [comment]
> But this is not practical.

Can you be more specific? I'm not sure what's not practical.

**Comment by sailsbot** [comment]
Thanks for posting, @listepo.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
